src/ring_theory/localization.lean

```diff
@@ -286,6 +286,18 @@ lemma mk'_eq_iff_eq {x₁ x₂} {y₁ y₂ : M} :
   f.mk' x₁ y₁ = f.mk' x₂ y₂ ↔ f.to_map (x₁ * y₂) = f.to_map (x₂ * y₁) :=
 f.to_localization_map.mk'_eq_iff_eq
 
+lemma mk'_mem_iff {x} {y : M} {I : ideal S} : f.to_map x ∈ I ↔ f.mk' x y ∈ I :=
```

Comment:
If the name is `mk'_mem_iff` then the statement should be `f.mk' x y ∈ I ↔ f.to_map x ∈ I`. Either the statement or the name should be changed so they match.